int16 UAV1
int16 UAV2
int16 UAV3
